vnext/local-cli/generator-windows/templates/cpp/src/MainPage.cpp

```diff
@@ -0,0 +1,24 @@
+﻿#include "pch.h"
+#include "MainPage.h"
+#if __has_include("MainPage.g.cpp")
+#include "MainPage.g.cpp"
+#endif
+
+#include "App.h"
+
+// clang-format off
+
+using namespace winrt;
+using namespace Windows::UI::Xaml;
+
+namespace winrt::<%=ns%>::implementation
+{
+    MainPage::MainPage()
+    {
+        InitializeComponent();
+        const auto& app = Application::Current().as<App>();
```

Comment:
>const auto& [](start = 8, length = 11)

Ideally we must not use references for temporaries.
Please change it to 'auto'.